interfaces/CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ if(HELICS_ENABLE_SWIG OR swig_required)
             set(
                 CMAKE_PROGRAM_PATH
                 "C:/ProgramData/chocolatey/bin"
+                "C:/local/swigwin-4.0.2"
```

Comment:
Is this just anticipating 4.0.2 coming out soon?

Reply:
For some reason I thought it was out already,  I will revert this change.  I think we do need the develop branch of swig to support octave 5.2 though as an aside